yt_dlp/extractor/tnaflix.py

```diff
@@ -92,9 +92,8 @@ def _real_extract(self, url):
         if not cfg_url:
             inputs = self._hidden_inputs(webpage)
             if inputs.get('vkey') and inputs.get('nkey'):
```

Comment:
apparently now some pages have the config url as a hidden input value, add a check for that
```suggestion
        inputs = self._hidden_inputs(webpage)
            
        if not cfg_url:
            cfg_url = inputs.get('config')

        if not cfg_url and inputs.get('vkey') and inputs.get('nkey'):
```
and fix indentation below